Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:bionic
+MAINTAINER devops@edx.org
+
+RUN apt-get update && apt-get upgrade -qy && apt-get install language-pack-en locales git python2.7 python-pip libmysqlclient-dev -qy && \
```

Comment:
Would be great to list each dependency and what it does / why we need it.

```
git; Used to pull in particular requirements from github rather than pypi
```

Reply:
Added